core/divesitehelpers.cpp

```diff
@@ -34,10 +34,10 @@ void ReverseGeoLookupThread::run()
 	QNetworkRequest request;
 	QNetworkAccessManager *rgl = new QNetworkAccessManager();
 	QEventLoop loop;
-	QString mapquestURL("http://open.mapquestapi.com/nominatim/v1/reverse.php?format=json&accept-language=%1&lat=%2&lon=%3");
+	//QString mapquestURL("http://open.mapquestapi.com/nominatim/v1/reverse.php?format=json&accept-language=%1&lat=%2&lon=%3");
```

Comment:
don't comment them out. just remove them. we had planned multiple services to lookup locations but that never went anywhere.

Reply:
ok